test/utils/setup.go

```diff
@@ -63,5 +64,13 @@ func SetupServiceTests(snapName string) (teardown func(), err error) {
 		return
 	}
 
+	// TODO: remove this patch which is for device services
+	if strings.Contains(snapName, "device-") {
+		err = injectDevicesAndProfilesDirConfig(strings.TrimPrefix(snapName, "edgex-"))
+		if err != nil {
+			log.Fatalf("Failed to inject devices/profiles dir into config: %s", err)
+		}
+	}
+
```

Comment:
Since the eKuiper suite has its own setup(), could you to make these changes on the [eKuiper side](https://github.com/canonical/edgex-snap-testing/blob/main/test/suites/ekuiper/main_test.go#L123) as well?